model/storage/RdsStorage.php

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016  (original work) Open Assessment Technologies SA;
+ *
+ * @author Alexander Zagovorichev <zagovorichev@1pt.com>
+ */
+
+namespace oat\taoEventLog\model\storage;
+
+use common_persistence_Manager;
+use common_persistence_Persistence;
+use common_persistence_SqlPersistence;
+use Doctrine\DBAL\Schema\AbstractSchemaManager;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\SchemaException;
+use oat\taoEventLog\model\StorageInterface;
+
+/**
+ * Class RdsStorage
+ * @package oat\taoEventLog\model\storage
+ */
+class RdsStorage implements StorageInterface
+{
+    const TABLE_NAME = 'event_log';
+    const OPTION_PERSISTENCE = 'persistence';
+
+    /**
+     * Persistence for DB
+     * @var common_persistence_Persistence
+     */
+    private $persistence;
+
+    /**
+     * RdsStorage constructor.
+     * @param string $persistence
+     */
+    public function __construct($persistence = '')
+    {
+        $this->persistence = $persistence;
+    }
+
+    /**
+     * @param string $eventName
+     * @param string $currentAction
+     * @param string $userIdentifier
+     * @param string $userRole
+     * @param string $occurred
+     * @param array $data
+     * @return bool
+     */
+    public function log($eventName = '', $currentAction = '', $userIdentifier = '', $userRole = '', $occurred = '', $data = [])
+    {
+        $result = $this->getPersistence()->insert(
+            self::TABLE_NAME, [
+                self::EVENT_NAME => $eventName,
+                self::ACTION => $currentAction,
+                self::USER_ID => $userIdentifier,
+                self::USER_ROLE => $userRole,
+                self::OCCURRED => $occurred,
+                self::PROPERTIES => json_encode($data)
+            ]
+        );
+
+        $id = $this->getPersistence()->lastInsertId(self::TABLE_NAME);
+
+        // todo clean data older than 90 days
```

Comment:
This is a blocking factor, could you move this logic to 90 days. Discuss with Tom, it's mandatory